CONTRIBUTING.md

```diff
@@ -8,6 +8,8 @@ Hi! Thanks for checking out Julia. If you have questions or problems, the [Julia
 
 * [Fork Julia](https://github.com/JuliaLang/julia/fork).
 
+* Sign up for [Travis](https://travis-ci.org/profile/) and switch on your julia fork.
```

Comment:
this is not really required for the majority of contributors - it doesn't hurt anything to do, but most users won't have osx testing activated for their account, for example